BLSQ/iaso
# POLIO-635: missing scope per round translation

iaso.polio.label.scope_per_round was not translated on scope tab while editing a campaign

## Self proof reading checklist

## Changes

Added json entries

## How to test
Open console
Go to campaigns, edit one, switch to scope tab and check that there is no warning displaying missing translation
